pkgs/by-name/sn/sncli/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  python312Packages,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  __Simperium3 = python312Packages.buildPythonPackage rec {
+    pname = "Simperium3";
+    version = "0.1.5";
+
+    src = python312Packages.fetchPypi {
+      inherit pname version;
+      sha256 = "0dyi2h0s3673kngns3rx8x7sv51bxi8dv0syv1gsssc8w5miif3q";
+    };
+
+    propagatedBuildInputs = with python312Packages; [ requests ];
+  };
```

Comment:
Please move this to `pkgs/development/python-modules/simperium3/default.nix` (refer to existing packages as examples) and add a `meta` section like below. This should be a separate commit.